### ISSUE TITLE
Parameter values aren't passed in during hyper-parameter search with concurrent builds

### ISSUE THREAD

**Issue by honnibal** [polyaxon/polyaxon#119]
When I run the following polyaxonfile:

```
---
version: 1

kind: group

settings:
  concurrency: 2
  random_search:
    n_experiments: 3
  matrix:
    param1:
      values: [1, 2]
    param2:
      values: [3, 4]

run:
  image: ubuntu
  cmd: /bin/bash run.sh
  build_steps:
    - apt-get update -y && apt-get install -y python3.6 python3-pip python3-distutils && python3 -m pip install polyaxon_helper
```

Where run.sh executes 

`python3 -c "import polyaxon_helper; import os; print(polyaxon_helper.get_declarations()); print(os.environ)"`

The hyper-parameters `param1` and `param2` are mapped to `None`. Without concurrency the values are mapped correctly.

Passing hyper-parameters in `env_vars` also doesn't work. I get a syntax error saying that the value is null if I do something like: `- [param1, {{ param1 }}]` in the `env_vars` list.

Edit: It seems passing the values as command-line arguments doesn't work either.

**Comment by mouradmourafiq** [comment]
Thanks for reporting, I will make a fix in the next few minutes. In the meantime, you can use this workaround `values: ['1', '2']`.

**Comment by honnibal** [comment]
Workaround works, thanks!

**Comment by mouradmourafiq** [comment]
The fix is part of the latest release 0.1.0
